library/gui/screen/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,42 @@
+{
+  "schemaVersion": 1,
+  "id": "quilt_screen",
+  "name": "Quilt Screen API",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/quilt_screen/icon.png",
+  "contact": {
+    "homepage": "https://quiltmc.org",
+    "issues": "https://github.com/QuiltMC/quilt-standard-libraries/issues",
+    "sources": "https://github.com/QuiltMC/quilt-standard-libraries"
+  },
+  "authors": [
+    "QuiltMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.12",
+    "minecraft": ">=1.18.2",
+    "quilt_base": "*"
+  },
+  "description": "Hooks and API to manipulate screens.",
+  "mixins": [
+    "quilt_screen.mixins.json"
+  ],
+  "custom": {
```

Comment:
It seems to be missing the injected interface stuff, you might want to look into the Tags API to see how it's done.

Reply:
Ah i used mixins instead of the special injected interface stuff, i switch to them now

Reply:
The implementation of the special injected interface stuff is done through mixins, but there's a special thing to tell loom that it's injected.

Reply:
Ah.

Reply:
Fixed.